app/solution_service.py

```diff
@@ -0,0 +1,13 @@
+from sympy.parsing.latex import parse_latex
+from sympy import symbols, solve
+
+def is_correct(question, attempt):
```

Comment:
This probably needs some thoughts around error handling specifically, but fine for now

Reply:
trying not to think about issues like near misses and what to do if we can't solve a problem

Reply:
updated this function to accept question id and question type as well, which will later be able to be used as hints for how sympy could validate the input